spec/controllers/admin/searchgov_domains_controller_spec.rb

```diff
@@ -1,21 +1,57 @@
 # frozen_string_literal: true
 
 describe Admin::SearchgovDomainsController do
-  fixtures :users, :searchgov_domains
-  let(:config) { described_class.active_scaffold_config }
+  fixtures :users, :searchgov_domains, :searchgov_urls
+
+  let(:basic_domain) { searchgov_domains(:basic_domain) }
+
+  before do
+    activate_authlogic
+    UserSession.create(users(:affiliate_admin))
+  end
+
+  describe 'after_create_save' do
+    it 'sets flash[:info] message correctly' do
+      expect { post :create, params: { record: { domain: 'www.usa.gov' } } }.
+        to change { SearchgovDomain.count }.by(1)
+
+      # Validate flash message
```

Comment:
This comment seems redundant to me as well and I'm not sure if it'll be helpful in the future. What do you think about removing it?